common/test/acceptance/tests/discussion/helpers.py

```diff
@@ -68,11 +69,11 @@ def add_manual_cohort(self, course_fixture, cohort_name):
         """
         Adds a cohort group by name, returning the ID for the group.
         """
-        url = LMS_BASE_URL + "/courses/" + course_fixture._course_key + '/cohorts/add'
-        data = {"name": cohort_name}
+        url = LMS_BASE_URL + "/courses/" + course_fixture._course_key + '/cohorts/'
+        data = json.dumps({"name": cohort_name})
```

Comment:
I'll try not doing json.dumps, but I don't think it worked without this. I'll let you know the details after I try it.